lib/ui/page/home/page/my_profile/controller.dart

```diff
@@ -194,41 +197,43 @@ class MyProfileController extends GetxController {
       }
     }
 
-    listInitIndex = router.profileSection.value?.index ?? 0;
-
-    bool ignoreWorker = false;
-    bool ignorePositions = false;
+    observerController.initialIndex = router.profileSection.value?.index ?? 0;
 
     _profileWorker = ever(
       router.profileSection,
       (ProfileTab? tab) async {
-        if (ignoreWorker) {
-          ignoreWorker = false;
-        } else {
-          ignorePositions = true;
-          await itemScrollController.scrollTo(
-            index: tab?.index ?? 0,
-            duration: 200.milliseconds,
-            curve: Curves.ease,
-          );
-          Future.delayed(Duration.zero, () => ignorePositions = false);
+        if (tab == _lastSelectedProfileTab) return;
 
-          highlight(tab);
-        }
+        _lastSelectedProfileTab = tab;
+        _isIgnoreObserving = true;
+
+        await observerController.animateTo(
+          index: tab?.index ?? 0,
+          duration: 200.milliseconds,
+          curve: Curves.ease,
+        );
+
+        highlight(tab);
       },
     );
 
-    positionsListener.itemPositions.addListener(() {
-      if (!ignorePositions) {
-        final ProfileTab tab = ProfileTab
-            .values[positionsListener.itemPositions.value.first.index];
-        if (router.profileSection.value != tab) {
-          ignoreWorker = true;
-          router.profileSection.value = tab;
-          Future.delayed(Duration.zero, () => ignoreWorker = false);
-        }
+    onObserve = (resultModel) {
```

Comment:
There seems to be a problem with keeping the selected profile section position when opening and closing the page. The position always changes, when non-first section is selected.

I have a video demonstration of that problem. On the left is the current `main` and on the right is the branch one:

https://github.com/user-attachments/assets/7f96b615-2e73-4dda-bb89-5c7a431ef722

Reply:
That's a problem connected with our conversation past this one, so information can be obtained there. 

I can try to correct this scenario for new behavior or just return an old one. What are your thoughts?

Reply:
I have returned old behavior

fixed on 3bdb4c364a9c19145ac3970c4389cdc034f8f30f commit